### DISCUSSION TITLE
How to auto disconnect the bot when music queue is empty

### DISCUSSION THREAD

**Discussion by Rayzan99** [Cog-Creators/Red-DiscordBot#5875] [Q&A] [answered]
Is there command for the bot to auto disconnect when the music queue is empty? the bot stays in voice channels for ages.

**Comment by Flame442** [comment, marked as answer]
Check out `[p]audioset dc` and `[p]audioset emptydisconnect`. For the future, support questions are better sent to our [support server](https://discord.gg/red).
